Content.Server/GameObjects/Components/Damage/RuinableComponent.cs

```diff
@@ -65,10 +69,11 @@ protected void PerformDestruction()
         {
             CurrentState = DamageState.Dead;
 
-            if (!Owner.Deleted && DestroySound != string.Empty)
+            if (!Owner.Deleted && DestroySounds.Count > 0)
             {
                 var pos = Owner.Transform.Coordinates;
-                EntitySystem.Get<AudioSystem>().PlayAtCoords(DestroySound, pos);
+                EntitySystem.Get<AudioSystem>()
+                    .PlayAtCoords(DestroySounds.Count == 1 ? DestroySounds[0] : _random.Pick(DestroySounds), pos);
```

Comment:
Soundcollection. And maybe use AudioHelpers to give this a random pitch? (0.125 approx)